lib/structures/Command.js

```diff
@@ -1,6 +1,18 @@
 "use strict";
+const Precondition = require("./Precondition");
+const Middleware = require("./Middleware");
```

Comment:
Why are you requiring Precondition and Middleware?

Reply:
This was because I was under the impression that `options.preconditions` & `options.middleware` would be `Precondition[]` & `Middleware[]` respectively.
Fixed.